### DISCUSSION TITLE
live del viernes 30

### DISCUSSION THREAD

**Discussion by YerValMoCa** [Launch-X-Latam/MisionFrontEnd#115] [Q&A]
saben si subirán el live de ayer viernes 30 de cómo usar figma y miro, llegue tarde, interesado en ver todo de nuevo

**Comment by maxkario** [comment]
Todos los live se suben una ver termina la sesion.

**Comment by CarloAyala** [comment]
Todavía no lo suben, puedes acceder con este links:[Sesión](https://web.microsoftstream.com/video/41b9721d-cde1-4ffd-bdf0-7a877d57e5e1)

**Comment by samujm** [comment]
En internet existe mucha documentación sobre figma, es muy amplia la comunidad.
Pero por experiencia propia te recomiento este curso para figma:
https://www.youtube.com/watch?v=_c26Fyi7RFA&list=PLvq-jIkSeTUbpfewvbKLhHctdkgadAy-K&ab_channel=jonmircha

y para miro estoy viendo este video apenas, aunque igual existen muchos en youtube
https://www.youtube.com/watch?v=cVaiM67oOOc&ab_channel=EDteam

Espero te sirva. Saludos.
